### PR TITLE
[CX-2071]: Add clickhouse to stack

### PR DESCRIPTION
**Pull request by gtoonstra**

### CODE REVIEW

File: charts/datafold/charts/clickhouse/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: clickhouse
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "22.8.4.7-df5"
```
Comment (gtoonstra) [inline charts/datafold/charts/clickhouse/Chart.yaml:6]: At some point this should probably become a setting elsewhere. Not sure how to deal with it yet

File: charts/datafold/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: datafold
 description: Helm chart package to deploy Datafold on kubernetes.
 type: application
-version: 0.1.2
```
Comment (cfernhout) [inline charts/datafold/Chart.yaml:5]: Do we automate this already?

**Review by cfernhout — APPROVED** [review]

File: charts/datafold/Chart.yaml (hunk 2 above)
Comment (gtoonstra) [reply charts/datafold/Chart.yaml:5]: @cfernhout : It's manual. Bit boring, but the linter checks it for us. Not sure how to automate this correctly tbh. You mean a pre-commit?

Comment (gtoonstra) [reply charts/datafold/Chart.yaml:5]: https://linear.app/datafold/issue/CX-2082/todo-at-end-of-milestone

**Pull request merged**